### PR TITLE
Update plugin docs formatting

### PR DESCRIPTION
**Pull request by kneal**

### CODE REVIEW

File: content/plugins/Getting started/how-to.md [hunk 1]
```diff
@@ -3,5 +3,7 @@ title: "How To"
 date: 2017-01-05
 weight: 2
 description: >
-  Coming Soon! How to build your own plugin.
----
\ No newline at end of file
+  Learn how to write plugin
```
Comment (meinenec) [inline content/plugins/Getting started/how-to.md]: "write a plugin" or "write plugins"?

File: content/plugins/Getting started/what-is.md [hunk 2]
```diff
@@ -3,6 +3,7 @@ title: "What Is"
 date: 2017-01-05
 weight: 1
 description: >
-  As stated previously Vela plugins are Docker containers designed to execute a specific set of instructions. Plugins can be written in any language and are recommended to be placed in a scratch container.
+  Learn about plugin
```
Comment (meinenec) [inline content/plugins/Getting started/what-is.md]: plugins?

File: content/plugins/_index.md [hunk 3]
```diff
@@ -4,6 +4,8 @@ title: "Plugins"
 linkTitle: "Plugins"
 weight: 20
 layout: plugins
+description: >
+  Here you can find information describing what are plugins and examples on how to use them
```
Comment (meinenec) [inline content/plugins/_index.md]: I think this should be "what plugins are"

**Review by meinenec — CHANGES_REQUESTED** [review]

File: content/plugins/_index.md (hunk 3 above)
Comment (kneal) [reply content/plugins/_index.md]: had that then reversed it. I can put it back. Both sound a little weird to me

File: content/plugins/Getting started/what-is.md (hunk 2 above)
Comment (kneal) [reply content/plugins/Getting started/what-is.md]: sure

File: content/plugins/Getting started/how-to.md (hunk 1 above)
Comment (kneal) [reply content/plugins/Getting started/how-to.md]: sure

**Review by meinenec — APPROVED** [review]

**Pull request merged**